lemming.go

```diff
@@ -56,9 +61,38 @@ func registerTestTask(gnmiServer *gnmit.GNMIServer, targetName string) error {
 	return gnmiServer.RegisterTask(testagentlocal.InterfaceTask(targetName))
 }
 
+// startSysrib starts the sysrib gRPC service at a unix domain socket. This
+// should be started prior to routing services to allow them to connect to
+// sysrib during their initialization.
+func startSysrib() {
+	if err := os.RemoveAll(sysrib.SockAddr); err != nil {
+		log.Fatal(err)
+	}
+
+	lis, err := net.Listen("unix", sysrib.SockAddr)
+	if err != nil {
+		log.Fatalf("listen error: %v", err)
+	}
+
+	grpcServer := grpc.NewServer()
+	s, err := sysrib.NewServer(nil)
+	if err != nil {
+		log.Fatalf("error while creating sysrib server: %v", err)
+	}
+	s.AddInterface("eth0", 0, true, "192.0.0.0/8", "DEFAULT")
```

Comment:
why this included?

Reply:
This was used for testing, I'll remove it.